src/app/containers/PageViewAnalytics/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { RequestContextConsumer } from '../../contexts/RequestContext';
+import Canonical from './Canonical';
+
+const ConsentBanner = props => (
```

Comment:
```suggestion
const PageViewAnalytics = props => (
```

This definitely isn't a ConsentBanner

Reply:
This fixes the dual `ConsentBanner` in the snapshots that I flagged above